pkg/api/persistentvolume/util_test.go

```diff
@@ -65,8 +65,8 @@ func TestPVSecrets(t *testing.T) {
 		{Spec: api.PersistentVolumeSpec{
 			ClaimRef: &api.ObjectReference{Namespace: "claimrefns", Name: "claimrefname"},
 			PersistentVolumeSource: api.PersistentVolumeSource{
-				RBD: &api.RBDVolumeSource{
-					SecretRef: &api.LocalObjectReference{
+				RBD: &api.RBDPersistentVolumeSource{
+					SecretRef: &api.SecretReference{
```

Comment:
Add a second PV spec that specifies a name and namespace in the RBD secretref (suggest namespace of "rbdns") and update expected namespaced secrets below

Reply:
done